python/sglang/srt/layers/rotary_embedding.py

```diff
@@ -147,7 +147,7 @@ def forward_cuda(
         key: torch.Tensor,
         offsets: Optional[torch.Tensor] = None,
     ) -> Tuple[torch.Tensor, torch.Tensor]:
-        if _is_cuda_available:
+        if _is_cuda_available and (self.head_size != 80):
```

Comment:
@lkm2835 Can you check if `/sgl-kernel/3rdparty/flashinfer/include/flashinfer/pos_enc.cuh:584` has specific divisibility regarding the RotaryEmbedding shape? Some kernels requires shape/size to be wholly divisible by a fixed amount. I want to see if this is better than `80` fix value patch which may not cover all fail cases.

Reply:
@Qubitium Thanks for replying.
I checked the FlashInfer code.
https://github.com/flashinfer-ai/flashinfer/blob/main/include/flashinfer/utils.cuh#L170-L197
Flashinfer only supports 64, 128, 256, 512.